src/App.test.js

```diff
@@ -1,8 +1,30 @@
-import { render, screen } from '@testing-library/react';
+import { render, screen, cleanup } from '@testing-library/react';
+import renderer from 'react-test-renderer';
+import { Provider } from 'react-redux';
 import App from './App';
+import store from './redux/store/configureStore';
 
-test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
+afterEach(() => cleanup());
+
+test('renders Rockets', () => {
+  render(
+    <Provider store={store}>
+      <App />
+    </Provider>,
+  );
+  const linkElement = screen.getByText(/Rockets/i);
   expect(linkElement).toBeInTheDocument();
 });
+
+const APP = (
+  <Provider store={store}>
+    <App />
+  </Provider>
+);
+
+describe('App Renders test', () => {
+  it('Renders App', () => {
+    const app = renderer.create(APP).toJSON();
+    expect(app).toMatchSnapshot();
+  });
```

Comment:
- It is good that you have written some tests, these are not enough to assert the proper functionality of different parts of the app. 
- [ ] Please consider adding tests for other components too.
- [ ] All pure redux functions should be tested. These include functions for fetching data i.e. rockets and missions.
- [ ] Consider testing for user interactions i.e. clicking on links in the Nav to navigate the app.

Reply:
Done.